tests/test_live_messages.py

```
@@ -284,3 +284,123 @@ async def __aexit__(self, exc_type, exc, tb):
+                pandascore_id=20,
+                team1="T1",
+                team2="GEN",
+                status="not_started",
+                game="league-of-legends",
+                scheduled_time=datetime.now(timezone.utc) + timedelta(hours=1),
+            )
+        )
+        session.commit()
+
+    class _ResultWrapper:
+        def __init__(self, rows):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)